src/main/java/net/sf/rails/ui/swing/StatusWindow.java

```diff
@@ -386,6 +386,26 @@ public void componentResized(ComponentEvent e) {
             }
         });
 
+        if ( Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.APP_QUIT_HANDLER) ) {
+            Desktop.getDesktop().setQuitHandler((e, r) -> {
+                if ( JOptionPane.showConfirmDialog(frame, LocalText.getText("CLOSE_WINDOW"), LocalText.getText("Select"), JOptionPane.OK_CANCEL_OPTION) == JOptionPane.OK_OPTION ) {
+                    frame.dispose();
+                    guiMgr.terminate();
+                    r.performQuit();
+                }
+                else {
+                    r.cancelQuit();
+                }
+            });
+        }
+
+//        SwingUtilities.invokeLater(() -> {
+//            JFrame main = new JFrame("java.awt.Desktop");
+//            main.setSize(new Dimension(600, 400));
+//            main.setLocationRelativeTo(null);
+//            main.setVisible(true);
+//        });
```

Comment:
Can this be removed?

Reply:
Did you see my first comment in the PR? I'd prefer not to remove this until you and @neutronc are able to verify this removal doesn't break the UI.

Reply:
I'll nuke it anyway-- if we need to restore that is what we have version control for